src/components/Questionnaire/Questionnaire.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import "./Questionnaire.css"
+
+class Questionnaire extends Component {
+  constructor() {
+    super()
+    // I assigned default values here in case a user decides not to move any/all sliders before clicking the submit button.
+    this.state = {
+      size: '3',
+      family_rating: '3',
+      trainability: '3',
+      grooming_needed: '3',
+      energy_level: '3'
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value })
+  }
+
+  searchBreeds = (event) => {
+    event.preventDefault()
+    const newBreedSearch = {
+      size: this.state.size,
+      family_rating: this.state.family_rating,
+      trainability: this.state.trainability,
+      grooming_needed: this.state.grooming_needed,
+      energy_level: this.state.energy_level
```

Comment:
```suggestion
  handleFormSubmit = (event) => {
    event.preventDefault()
    const newBreedSearch = {
      size: this.state.size,
      family_rating: this.state.family_rating,
      trainability: this.state.trainability,
      grooming_needed: this.state.grooming_needed,
      energy_level: this.state.energy_level
```

Reply:
Just a small name suggestion, since our api call will be what makes the search and this is the method that handles the form submission.